### ISSUE TITLE
Design way of showing URL

### ISSUE THREAD

**Issue by mallikarai05** [tijlleenders/ZinZen#491]

**Comment by mallikarai05** [comment]
![image](https://user-images.githubusercontent.com/73054648/175820954-6102f739-ab6f-48ab-94e8-cf43441b2586.png)
@tijlleenders as discussed, We'll have one input field only and URLs will be recognized as magic text. Will this be okay?

**Comment by tijlleenders** [comment]
Yeah, great!

**Comment by tijlleenders** [comment]
The easiest way to implement is probably activating the link not on the text but on the > - right?

If that's easier then fine. Don't think the > should be shown if there are no subGoals anyway.

Let's try it for a while - we can always change / adjust.

**Comment by tijlleenders** [comment]
Activating on the text would also prevent you from editing the URL.

**Comment by mallikarai05** [comment]
Right, so then '>' is to be removed from the design, also it will help in determining if there are subgoals or not.

**Comment by tijlleenders** [comment]
Thanks!
Offtopic: For subgoals we could also do something with the design of the whole card. 
Like a deck of cards to show there are underlying cards.

**Comment by mallikarai05** [comment]
> Thanks! Offtopic: For subgoals we could also do something with the design of the whole card. Like a deck of cards to show there are underlying cards.

That sounds cool, if it takes a lot of space we can also go for box-shadow? 
I'll try it out
